modules/references/client/components/create-reference/Feedback.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export default function Feedback({ feedback, onChangeFeedback }) {
+  const { t } = useTranslation('references');
+
+  return (
+    <div className="panel panel-default">
+      <div className="panel-heading">
+        <h4 id="feedback-public-question">
+          {t('Would you like to describe something about them?')}
+        </h4>
+      </div>
+      <div
+        className="panel-body"
+        role="group"
+        aria-labelledby="feedback-public-question"
+      >
+        <label htmlFor="feedback-message" className="control-label">
+          {t('Public feedback')}
```

Comment:
Non-blocking:

It's not clear to me that one can/should skip this step. Perhaps adding `(optional)` here or somewhere would help communicate our intention: `"Speak only if you have something to say"`.
WDYT?

Reply:
"(Optional)" seems like a great quick way to make it clearer 👍